### ISSUE TITLE
Calystegia pellita - Exclude from Ontario

### ISSUE THREAD

**Issue by MichaelOldham** [Canadensys/vascan-data#2425]
http://data.canadensys.net/vascan/taxon/4609?lang=en

Newmaster et al. (1998) is listed in VASCAN as the source for Ontario for both Calystegia hederacea and C. pellita, though Newmaster et al. (1998) list only C. pubescens (a synonym for C. hederacea in VASCAN).  However, Newmaster et al. (1998) list C. pellita as a synonym for C. pubescens. So a single species entry in Newmaster et al. is being used as the source for two different species in VASCAN.

From what I have been able to determine there are valid Ontario records only for C. hederacea (C. pubescens if distinguished from C. hederacea), including a specimen in the NHIC herbarium which is the primary basis for inclusion in Newmaster et al.  I know of no valid Ontario records for C. pellita (or C. hederacea sensu stricto).  

There is no indication of which taxonomic treatment was used for the VASCAN Calystegia treatment and this group is treated differently in different floras.  The Michigan Field Manual (Voss & Reznicek 2012) follows the recent Flora of China treatment (http://www.efloras.org/florataxon.aspx?flora_id=2&taxon_id=105355) which accepts C. hederacea, C. pellita, and C. pubescens as three distinct species.  Using that key I arrive at C. pubescens (i.e. C. hederacea of VASCAN) for the specimen in the NHIC herbarium (originally determined as Convolvulus japonicus cv. Flore Pleno, but collected as a weed along a railroad bed in Essex County, Ontario).  Note that the Michigan Field Manual and Flora Novae Angliae (Haines 2011) only list C. pubescens for Michigan and New England, respectively, and not C. hederacea or C. pellita.

MIKE OLDHAM

**Comment by brouille** [comment]
Calystegia pellita is listed in FOIBIS, however, with C. pub. and jap. as syn
I have been waiting for the FNA treatment to have a better grasp of issues here.

**Comment by brouille** [comment]
http://www.uoguelph.ca/foibis/detail_2.cfm?type=latin&genus=Calystegia&species=pellita&oplid=1321

**Comment by brouille** [comment]
The late Richard Brummitt did not deal with C. pellita in his treatment for FNA.
However, given the confusion resulting from the use of several names for C. pubescens in NA, following Fl. Michigan online, I excluded C. pellita from Canada.
Under C. pubescens, I added as misapplied synonyms (auct. non) : C. pellita and C. hederacea.

This should solve the issue.

Regrettably, despite molecular phylogenetic data to the contrary, Brummitt chose to segregate Calystegia from Convolvulus in which it is embedded. This will need a closer look later.
LB
